# Consolidated Western Wear Retailers: Regression Analysis to Understand Cost Drivers in a Purchasing Department

based on the case study by Seargeant, A. M. A. 2017. IMA Educational Case Journal, 10 (4), pp. 1-3.

## Table of Contents
* [Introduction](#Introduction)
* [The Problem](#The-Problem)
* [Requirements](#Requirements)
* [Import Modules and Data](#Import-Modules-and-Data)
* [Statistical Analysis of the Costs](#Statistical-Analysis-of-the-Costs)
 * [Scatter Plots](#Scatter-Plots)
 * [Exclude Unusual Events](#Exclude-Unusual-Events)
 * [Regression Analysis](#Regression-Analysis)
 * [Choice of the Best Model](#Choice-of-the-Best-Model)
 * [Meaning of the Model from an Economic Perspective](#Meaning-of-the-Model-from-an-Economic-Perspective)
* [Recommendations](#Recommendations)
 * [Number of Suppliers](#Number-of-Suppliers)
 * [Number of Purchase Orders](#Number-of-Purchase-Orders)

## Introduction

Dan’s Western Wear began its humble roots on Main Street
in Sheridan, Wyoming, where Thomas George, a Greek
immigrant, settled his family in 1919 and began a cobbler
business that served ranchers, cowboys, and coal miners. Dan
George, Thomas’ son, was nine years old when they arrived
in the United States and was soon active in his father’s
cobbler business. Ultimately, he took over the business and
began to increase the store’s inventory, creating the business
that is known today as Dan’s Western Wear, which sells
quality boots, work clothes, western apparel, and jewelry to
just about anybody. Dan believes that if clothes can hold up
to a dusty hard day’s work, and look great at the same time,
all the better. The company prides itself on quality clothes
with good old-fashioned service. Customers are greeted at
the door and helped throughout their experience. A visit to
Dan’s Western Wear is like a visit to an old friend.

Dan’s Western Wear is typical of western wear retail
outlets. The industry is characterized by many independent
regional shops. Frequently, these retail shops have been
in operation for many years and have a multigenerational,
loyal customer base. Recently, the online western wear
business has shown some growth, but customers are
primarily interested in face-to-face shopping where they
can feel the products, try them on, and visit with locals who
share their interests. Because these specialty retail outlets
have relatively little competition from online and discount
retailers, western wear stores have enjoyed relatively high
profit margins for retail businesses. If managed well, these
can be profitable stores.

Joe West, a young entrepreneur who had worked at
Dan’s Western Wear prior to going to graduate school for a
master’s in business administration, saw an opportunity to
consolidate western wear retail outlets to gain efficiencies.
Thus, he founded Consolidated Western Wear Retailers
(CWWR) and began purchasing profitable western wear
stores, starting with Dan’s Western Wear. He also assembled
a management team to help him grow his business. As a
new store is purchased, Joe and the management team study
the store to learn how it is successful. Initially, they do not
make major changes, keeping the same store name and
encouraging the store to operate as before. Over time, Joe
and the management team implement changes to improve
profitability. Currently, CWWR owns 21 retail stores, and
Joe believes this is large enough to aggressively pursue
efficiencies from consolidation.

[[Back to TOC]](#Table-of-Contents)

## The Problem

Each year Joe reviews the financial information for all the CWWR stores. This past year was a relatively good year; company profits were up despite the huge July Fourth fire in Las Vegas, Nevada, that shut down the store for four months and required replacement of all the inventory. Joe did notice, however, that purchasing department costs varied considerably between stores. The minimum was 575,000 USD, and the maximum was 2.2 million USD. This was perplexing, and he thought this might be an area where efficiencies could be achieved. Currently, each store has its own purchasing department with full autonomy. In the western wear industry, regional customers have regional tastes and desires. Local purchasing agents are thought to be best able to understand the desires of local customers and to meet those needs.

On his management team, Joe has a managerial cost specialist with skills in data analytics. Together they discussed the purchasing department cost problem and identified three potential cost drivers: merchandise purchased, number of purchase orders, and number of suppliers. To verify these ideas, Joe contacted purchasing managers from three different stores who agreed that these were potentially good cost drivers and that no others were readily apparent. The managerial cost specialist gathered data for the four variables from last year’s financial information. The Excel-file 'Consolidated Western Wear Retailers Data.xlsx' contains the data.

[[Back to TOC]](#Table-of-Contents)

## Requirements

Joe asked you, the managerial cost specialist on his management team, to examine the data and to recommend some courses of action to reduce purchasing department costs. Use the following requirements as a guide:

**Prepare a statistical analysis of the costs provided:**

1. Plot the purchasing department cost vs. each cost driver.
2. Analyze the data for potential problems, correct data problems if necessary, and report any changes made.
3. Use regression analysis to develop cost models for all potential cost drivers.
4. Identify the best model, and explain why.
5. Explain what the model means from an economic perspective.

**Use the model to make two recommendations to the CWWR management team for improving the efficiency of purchasing operations:**

6. Be specific with details of the recommendations.
7. Estimate the cost savings from the implementation of your recommendations.
8. Consider secondary implications, quantitative and/or qualitative.
9. Indicate how these changes (recommendations) should be implemented.

[[Back to TOC]](#Table-of-Contents)

## Import Modules and Data

In [1]:
import statsmodels.formula.api as smf 
import pandas as pd
import plotly.graph_objects as go
from plotly.subplots import make_subplots
import numpy as np

In [5]:
df = pd.read_excel(r'Consolidated Western Wear Retailers Data.xlsx')
df.head(10)

#Merchandise Purchase: Each item you order
#Purchase orders: How often you orders? 

StoreLocation  PurchasingDeptCost  MerchandisePurchased  PurchaseOrders  \
0    Sheridan, WY              575000              47239000            1708   
1          Denver             1226000             102364000            2519   
2  Salt Lake City             1710000             100162000            2506   
3     Kansas City              881000              95760000            1719   
4           Omaha             1544000              51466000            2883   
5       Milwaukee              794000              50631000             647   
6     Minneapolis             1341000              84753000            2978   
7         Phoenix              794000             103464000            3761   
8       Las Vegas             2216000              96162000            2584   
9     Albuquerque             2030000              62364000            5497   

   Suppliers  
0         61  
1         95  
2        139  
3         91  
4        155  
5         75  
6        103  
7        117  
8         73  
9        176

[[Back to TOC]](#Table-of-Contents)

## Statistical Analysis of the Costs

### Scatter Plots

In [6]:
fig = make_subplots(rows=3, cols=1)

fig.add_trace(
    go.Scatter(x=df['MerchandisePurchased'], 
               y=df['PurchasingDeptCost'],
               mode='markers', text=df['StoreLocation'] ),

1,1)

fig.add_trace(
    go.Scatter(x=df['PurchaseOrders'], 
               y=df['PurchasingDeptCost'], mode='markers', text=df['StoreLocation']),
2,1)


fig.add_trace(
    go.Scatter(x=df['Suppliers'], 
               y=df['PurchasingDeptCost'], mode='markers', text=df['StoreLocation']),
3,1)

fig.update(layout_showlegend=False)


fig.update_layout(
    xaxis=dict( title='Merchandise Purchased ($)'),
    xaxis2=dict( title='Number of Purchase Orders'),
    xaxis3=dict( title='Number of Suppliers'),
    yaxis=dict( title = 'Purchasing Dept. Cost ($)'),
    yaxis2=dict( title = 'Purchasing Dept. Cost ($)'),
    yaxis3=dict( title = 'Purchasing Dept. Cost ($)')
)



fig.update_layout(
    height=800,
    title_text="Scatter Plots For Each Cost Driver"
    )


        
fig.show()

[[Back to TOC]](#Table-of-Contents)

### Exclude Unusual Events

Especially the scatter plots for the number of purchase orders and the number of suppliers show that the Las Vegas store appears to be an outlier. Based on the setting, we also know there was a huge July Fourth fire, and the store had to be shut down for four months, and all the inventory had to be replaced. Because this is an unusual event that is unlikely to occur again, we eliminate this observation from the further analysis.

In [7]:
df = df[df['StoreLocation'] != 'Las Vegas']

[[Back to TOC]](#Table-of-Contents)

### Regression Analysis

In the following, we estimate several regression model specifications to determine the model that is best at explaining the purchasing department costs.

In [8]:
model = smf.ols('PurchasingDeptCost ~ MerchandisePurchased', data=df)
model = model.fit()
print(model.summary())

                            OLS Regression Results                            
Dep. Variable:     PurchasingDeptCost   R-squared:                       0.009
Model:                            OLS   Adj. R-squared:                 -0.046
Method:                 Least Squares   F-statistic:                    0.1689
Date:                Mon, 21 Mar 2022   Prob (F-statistic):              0.686
Time:                        14:07:09   Log-Likelihood:                -284.58
No. Observations:                  20   AIC:                             573.2
Df Residuals:                      18   BIC:                             575.2
Df Model:                           1                                         
Covariance Type:            nonrobust                                         
                           coef    std err          t      P>|t|      [0.025      0.975]
----------------------------------------------------------------------------------------
Intercept             1.033e+06 

In [9]:
model = smf.ols('PurchasingDeptCost ~ PurchaseOrders', data=df)
model = model.fit()
print(model.summary())

                            OLS Regression Results                            
Dep. Variable:     PurchasingDeptCost   R-squared:                       0.368
Model:                            OLS   Adj. R-squared:                  0.332
Method:                 Least Squares   F-statistic:                     10.46
Date:                Mon, 21 Mar 2022   Prob (F-statistic):            0.00460
Time:                        14:07:11   Log-Likelihood:                -280.09
No. Observations:                  20   AIC:                             564.2
Df Residuals:                      18   BIC:                             566.2
Df Model:                           1                                         
Covariance Type:            nonrobust                                         
                     coef    std err          t      P>|t|      [0.025      0.975]
----------------------------------------------------------------------------------
Intercept        7.33e+05   1.39e+05      5.

In [10]:
model = smf.ols('PurchasingDeptCost ~ Suppliers', data=df)
model = model.fit()
print(model.summary())

                            OLS Regression Results                            
Dep. Variable:     PurchasingDeptCost   R-squared:                       0.516
Model:                            OLS   Adj. R-squared:                  0.489
Method:                 Least Squares   F-statistic:                     19.18
Date:                Mon, 21 Mar 2022   Prob (F-statistic):           0.000361
Time:                        14:07:12   Log-Likelihood:                -277.42
No. Observations:                  20   AIC:                             558.8
Df Residuals:                      18   BIC:                             560.8
Df Model:                           1                                         
Covariance Type:            nonrobust                                         
                 coef    std err          t      P>|t|      [0.025      0.975]
------------------------------------------------------------------------------
Intercept   4.343e+05   1.69e+05      2.573      0.0

In [9]:
model = smf.ols('PurchasingDeptCost ~ MerchandisePurchased + PurchaseOrders', data=df)
model = model.fit()
print(model.summary())

                            OLS Regression Results                            
Dep. Variable:     PurchasingDeptCost   R-squared:                       0.375
Model:                            OLS   Adj. R-squared:                  0.301
Method:                 Least Squares   F-statistic:                     5.100
Date:                Fri, 18 Mar 2022   Prob (F-statistic):             0.0184
Time:                        15:30:46   Log-Likelihood:                -279.97
No. Observations:                  20   AIC:                             565.9
Df Residuals:                      17   BIC:                             568.9
Df Model:                           2                                         
Covariance Type:            nonrobust                                         
                           coef    std err          t      P>|t|      [0.025      0.975]
----------------------------------------------------------------------------------------
Intercept             6.519e+05 

In [10]:
model = smf.ols('PurchasingDeptCost ~ MerchandisePurchased  + Suppliers', data=df)
model = model.fit()
print(model.summary())

                            OLS Regression Results                            
Dep. Variable:     PurchasingDeptCost   R-squared:                       0.519
Model:                            OLS   Adj. R-squared:                  0.462
Method:                 Least Squares   F-statistic:                     9.161
Date:                Fri, 18 Mar 2022   Prob (F-statistic):            0.00200
Time:                        15:30:49   Log-Likelihood:                -277.36
No. Observations:                  20   AIC:                             560.7
Df Residuals:                      17   BIC:                             563.7
Df Model:                           2                                         
Covariance Type:            nonrobust                                         
                           coef    std err          t      P>|t|      [0.025      0.975]
----------------------------------------------------------------------------------------
Intercept             4.754e+05 

In [11]:
model = smf.ols('PurchasingDeptCost ~ PurchaseOrders + Suppliers', data=df)
model = model.fit()
print(model.summary())

                            OLS Regression Results                            
Dep. Variable:     PurchasingDeptCost   R-squared:                       0.684
Model:                            OLS   Adj. R-squared:                  0.647
Method:                 Least Squares   F-statistic:                     18.40
Date:                Fri, 18 Mar 2022   Prob (F-statistic):           5.59e-05
Time:                        15:30:52   Log-Likelihood:                -273.15
No. Observations:                  20   AIC:                             552.3
Df Residuals:                      17   BIC:                             555.3
Df Model:                           2                                         
Covariance Type:            nonrobust                                         
                     coef    std err          t      P>|t|      [0.025      0.975]
----------------------------------------------------------------------------------
Intercept       2.804e+05   1.49e+05      1.

In [12]:
model = smf.ols('PurchasingDeptCost ~ MerchandisePurchased + PurchaseOrders + Suppliers', data=df)
model = model.fit()
print(model.summary())

                            OLS Regression Results                            
Dep. Variable:     PurchasingDeptCost   R-squared:                       0.685
Model:                            OLS   Adj. R-squared:                  0.626
Method:                 Least Squares   F-statistic:                     11.60
Date:                Fri, 18 Mar 2022   Prob (F-statistic):           0.000274
Time:                        15:30:55   Log-Likelihood:                -273.12
No. Observations:                  20   AIC:                             554.2
Df Residuals:                      16   BIC:                             558.2
Df Model:                           3                                         
Covariance Type:            nonrobust                                         
                           coef    std err          t      P>|t|      [0.025      0.975]
----------------------------------------------------------------------------------------
Intercept             3.069e+05 

[[Back to TOC]](#Table-of-Contents)

### Choice of the Best Model

We can see that the multivariate regression model with the number of purchase orders and the number of suppliers as independent variables is the one with the highest adjusted R^2 and, therefore, should be used for further analysis.

**Final Model**

$$ Purchasing Department Cost = 2.804e+05 +  105.4219 * Number of Purhcase Orders + 5247.5090 * Number of Suppliers $$



[[Back to TOC]](#Table-of-Contents)

### Meaning of the Model from an Economic Perspective

The amount of merchandise purchased is a good indication of the size of the store. It is also a common allocation basis for a purchasing department. In general, it would seem economically plausible that the more merchandise purchased, the larger the purchasing department costs would be. But analysis of this data shows no relationship between these two variables.

The number of purchase orders measures the activity of ordering merchandise. Some stores order large amounts of merchandise per purchase order, whereas others make many small purchases. The more purchase orders, the more people are needed, and the higher the purchasing department costs. The analysis supports this relationship.

The number of suppliers measures the activity of maintaining suppliers. The more suppliers a store uses, the more supplier relations have to be maintained, and the more inventory has to be reviewed, recorded, and maintained. This should lead to higher purchasing department costs. The analysis also supports this relationship.


[[Back to TOC]](#Table-of-Contents)

## Recommendations

We use a conservative estimate based on the final regression model to estimate the potential cost savings from the recommended activities. We use 100 USD per purchase order and 5000 USD per supplier.

- Find activities that decreases two of the predictors. 

### Number of Suppliers

The number of suppliers should not depend on the amount of merchandise purchased. Currently, the range of suppliers is from 34 to 176, with a median of 111. Supplier reduction should initially be targeted at those with an extremely high number of suppliers. The recommendation is for all stores to reduce their suppliers to 125 or fewer. For all but two stores, this will be a reduction of 3% to 19% and should not be onerous. The two stores with the most suppliers would have to decrease their suppliers by almost 30%. Thus, the recommendation is as follows: all stores should reduce their number of suppliers to 125 or fewer within six months, except those with more than 150 suppliers; those stores should reduce their suppliers by 15% in the first six months and to 125 in the second six months. 


- It's the firm that decides who to cut as a suppliers. 

In [11]:
df['Suppliers'].describe()

count     20.00000
mean     108.00000
std       42.37303
min       34.00000
25%       71.75000
50%      111.00000
75%      139.50000
max      176.00000
Name: Suppliers, dtype: float64

In [13]:

fig = go.Figure(data=[go.Histogram(x=df['Suppliers'],
                                  xbins=dict(size=5.0),)])

fig.update_layout(title_text='Distribution of Number of Suppliers')
    
    
fig.show()

In [14]:
df['SuppliersReduction'] = df['Suppliers']-125
df.loc[df['SuppliersReduction']<0,'SuppliersReduction']=0
df['SuppliersReductionPerc']= df['SuppliersReduction']/df['Suppliers']*100
df

StoreLocation  PurchasingDeptCost  MerchandisePurchased  PurchaseOrders  \
0     Sheridan, WY              575000              47239000            1708   
1           Denver             1226000             102364000            2519   
2   Salt Lake City             1710000             100162000            2506   
3      Kansas City              881000              95760000            1719   
4            Omaha             1544000              51466000            2883   
5        Milwaukee              794000              50631000             647   
6      Minneapolis             1341000              84753000            2978   
7          Phoenix              794000             103464000            3761   
9      Albuquerque             2030000              62364000            5497   
10          Tucson             1338000              65635000            4347   
11         Houston              856000              88524000            2878   
12   Oklahoma City             1122000              72645000             819   
13           Tulsa              863000              61638000            1247   
14          Dallas             1085000             105666000            2162   
15     San Antonio              952000              59437000            2822   
16          Austin             1134000              38542000            5115   
17         El Paso             1042000              33020000             382   
18       Nashville             1634000              36322000            5293   
19         Memphis              699000              34121000             967   
20    Indianapolis              875000              31920000            2425   

    Suppliers  SuppliersReduction  SuppliersReductionPerc  
0          61                   0                0.000000  
1          95                   0                0.000000  
2         139                  14               10.071942  
3          91                   0                0.000000  
4         155                  30               19.354839  
5          75                   0                0.000000  
6         103                   0                0.000000  
7         117                   0                0.000000  
9         176                  51               28.977273  
10        130                   5                3.846154  
11         62                   0                0.000000  
12        129                   4                3.100775  
13        145                  20               13.793103  
14        141                  16               11.347518  
15        105                   0                0.000000  
16         51                   0                0.000000  
17        131                   6                4.580153  
18        172                  47               27.325581  
19         34                   0                0.000000  
20         48                   0                0.000000

In [15]:
np.sum(df['SuppliersReduction']>0)

9

In [16]:
df['SuppliersReduction1'] = df['SuppliersReduction']
df.loc[df['Suppliers']>150,'SuppliersReduction1']=round(df['Suppliers']*0.15)
df

StoreLocation  PurchasingDeptCost  MerchandisePurchased  PurchaseOrders  \
0     Sheridan, WY              575000              47239000            1708   
1           Denver             1226000             102364000            2519   
2   Salt Lake City             1710000             100162000            2506   
3      Kansas City              881000              95760000            1719   
4            Omaha             1544000              51466000            2883   
5        Milwaukee              794000              50631000             647   
6      Minneapolis             1341000              84753000            2978   
7          Phoenix              794000             103464000            3761   
9      Albuquerque             2030000              62364000            5497   
10          Tucson             1338000              65635000            4347   
11         Houston              856000              88524000            2878   
12   Oklahoma City             1122000              72645000             819   
13           Tulsa              863000              61638000            1247   
14          Dallas             1085000             105666000            2162   
15     San Antonio              952000              59437000            2822   
16          Austin             1134000              38542000            5115   
17         El Paso             1042000              33020000             382   
18       Nashville             1634000              36322000            5293   
19         Memphis              699000              34121000             967   
20    Indianapolis              875000              31920000            2425   

    Suppliers  SuppliersReduction  SuppliersReductionPerc  SuppliersReduction1  
0          61                   0                0.000000                  0.0  
1          95                   0                0.000000                  0.0  
2         139                  14               10.071942                 14.0  
3          91                   0                0.000000                  0.0  
4         155                  30               19.354839                 23.0  
5          75                   0                0.000000                  0.0  
6         103                   0                0.000000                  0.0  
7         117                   0                0.000000                  0.0  
9         176                  51               28.977273                 26.0  
10        130                   5                3.846154                  5.0  
11         62                   0                0.000000                  0.0  
12        129                   4                3.100775                  4.0  
13        145                  20               13.793103                 20.0  
14        141                  16               11.347518                 16.0  
15        105                   0                0.000000                  0.0  
16         51                   0                0.000000                  0.0  
17        131                   6                4.580153                  6.0  
18        172                  47               27.325581                 26.0  
19         34                   0                0.000000                  0.0  
20         48                   0                0.000000                  0.0

In [17]:
print('In the first 6 months, the number of suppliers is reduced by', np.sum(df['SuppliersReduction1']),'.')
print('This results in estimated cost savings of'  ,np.sum(df['SuppliersReduction1'])*5000, 'USD per year.')

print('After 12 months, suppliers will be reduced by', np.sum(df['SuppliersReduction']),'in total.')
print('A conservative estimate of the total cost savings from reducing suppliers is'  ,np.sum(df['SuppliersReduction'])*5000, 'USD per year.')

In the first 6 months, the number of suppliers is reduced by 140.0 .
This results in estimated cost savings of 700000.0 USD per year.
After 12 months, suppliers will be reduced by 193 in total.
A conservative estimate of the total cost savings from reducing suppliers is 965000 USD per year.


DO SOME RECOMMENDATIONS!!!

[[Back to TOC]](#Table-of-Contents)

### Number of Purchase Orders

As stores increase in size, the number of purchase orders is expected to increase. Thus, a better metric for analyzing purchase order activity is the average merchandise purchased per purchase order (MP/PO). Across all stores, the MP/PO ranged from 6,862 USD to 88,699 USD. Two stores were particularly low (under 10,000 USD), and four more were somewhat underachieving (under 20,000 USD).

- We want high numbers of purchase orders because it means we have less inventory(?)

In [29]:
df['MPPO']=df['MerchandisePurchased']/df['PurchaseOrders']
df['MPPO'].describe()

count       20.000000
mean     36529.955737
std      25002.660048
min       6862.270924
25%      17163.387347
50%      29609.282410
75%      49012.900341
max      88699.633700
Name: MPPO, dtype: float64

In [30]:
fig = go.Figure(data=[go.Histogram(x=df['MPPO'],
                                  xbins=dict(size=10000.0),)])

fig.update_layout(title_text='Distribution of Merchandise Purchased per Purchase Order')
    
    
fig.show()

In [31]:
np.sum(df['MPPO']<20000)

6

In [32]:
np.sum(df['MPPO']<10000)

2

The target for all stores is to be more than 20,000 USD of merchandise per purchase order. In order to implement this change reasonably, stores should increase their average MP/PO by 15% every 4 months if the MP/PO rate is less than the 20,000 USD rate and 20% if less than the 10,000 USD rate.

In [33]:
df['Orders1'] = df['PurchaseOrders']
df.loc[df['MPPO']<20000,'Orders1']=round(df['MerchandisePurchased']/(df['MPPO']*1.15))
df.loc[df['MPPO']<10000,'Orders1']=round(df['MerchandisePurchased']/(df['MPPO']*1.2))
df['OrdersRed1'] = df['PurchaseOrders']-df['Orders1']
df['MPPO1']=df['MerchandisePurchased']/df['Orders1']


df['Orders2'] = df['Orders1']
df.loc[df['MPPO1']<20000,'Orders2']=round(df['MerchandisePurchased']/(df['MPPO1']*1.15))
df.loc[df['MPPO1']<10000,'Orders2']=round(df['MerchandisePurchased']/(df['MPPO1']*1.2))
df['OrdersRed2'] = df['Orders1']-df['Orders2']
df['MPPO2']=df['MerchandisePurchased']/df['Orders2']

df['Orders3'] = df['Orders2']
df.loc[df['MPPO2']<20000,'Orders3']=round(df['MerchandisePurchased']/(df['MPPO2']*1.15))
df.loc[df['MPPO2']<10000,'Orders3']=round(df['MerchandisePurchased']/(df['MPPO2']*1.2))
df['OrdersRed3'] = df['Orders2']-df['Orders3']
df['MPPO3']=df['MerchandisePurchased']/df['Orders3']

In [34]:
df['FinalOrderTarget'] = df['PurchaseOrders']
df.loc[df['MPPO']<20000,'FinalOrderTarget']=np.floor(df['MerchandisePurchased']/20000)

df['FinalOrderRed']= df['PurchaseOrders']  - df['FinalOrderTarget']


In [35]:
df.sort_values(by=['MPPO'])

StoreLocation  PurchasingDeptCost  MerchandisePurchased  PurchaseOrders  \
18       Nashville             1634000              36322000            5293   
16          Austin             1134000              38542000            5115   
9      Albuquerque             2030000              62364000            5497   
20    Indianapolis              875000              31920000            2425   
10          Tucson             1338000              65635000            4347   
4            Omaha             1544000              51466000            2883   
15     San Antonio              952000              59437000            2822   
7          Phoenix              794000             103464000            3761   
0     Sheridan, WY              575000              47239000            1708   
6      Minneapolis             1341000              84753000            2978   
11         Houston              856000              88524000            2878   
19         Memphis              699000              34121000             967   
2   Salt Lake City             1710000             100162000            2506   
1           Denver             1226000             102364000            2519   
14          Dallas             1085000             105666000            2162   
13           Tulsa              863000              61638000            1247   
3      Kansas City              881000              95760000            1719   
5        Milwaukee              794000              50631000             647   
17         El Paso             1042000              33020000             382   
12   Oklahoma City             1122000              72645000             819   

    Suppliers  SuppliersReduction  SuppliersReductionPerc  \
18        172                  47               27.325581   
16         51                   0                0.000000   
9         176                  51               28.977273   
20         48                   0                0.000000   
10        130                   5                3.846154   
4         155                  30               19.354839   
15        105                   0                0.000000   
7         117                   0                0.000000   
0          61                   0                0.000000   
6         103                   0                0.000000   
11         62                   0                0.000000   
19         34                   0                0.000000   
2         139                  14               10.071942   
1          95                   0                0.000000   
14        141                  16               11.347518   
13        145                  20               13.793103   
3          91                   0                0.000000   
5          75                   0                0.000000   
17        131                   6                4.580153   
12        129                   4                3.100775   

    SuppliersReduction1          MPPO  Orders1  OrdersRed1         MPPO1  \
18                 26.0   6862.270924   4411.0       882.0   8234.413965   
16                  0.0   7535.092864   4262.0       853.0   9043.172220   
9                  26.0  11345.097326   4780.0       717.0  13046.861925   
20                  0.0  13162.886598   2109.0       316.0  15135.135135   
10                  5.0  15098.918795   3780.0       567.0  17363.756614   
4                  23.0  17851.543531   2507.0       376.0  20528.919027   
15                  0.0  21062.012757   2822.0         0.0  21062.012757   
7                   0.0  27509.704866   3761.0         0.0  27509.704866   
0                   0.0  27657.494145   1708.0         0.0  27657.494145   
6                   0.0  28459.704500   2978.0         0.0  28459.704500   
11                  0.0  30758.860320   2878.0         0.0  30758.860320   
19                  0.0  35285.418821    967.0         0.0  35285.418821   
2                  14.0  39968.874701   2506.0        

In [36]:
print('After the first 4 months, purchase orders are reduced by', round(np.sum(df['OrdersRed1'])), 'per year')
print('After 8 months, purchase orders are additionally reduced by', round(np.sum(df['OrdersRed2'])), 'per year')
print('After 1 year, purchase orders are additionally reduced by', round(np.sum(df['OrdersRed3'])), 'per year' )

After the first 4 months, purchase orders are reduced by 3711 per year
After 8 months, purchase orders are additionally reduced by 2836 per year
After 1 year, purchase orders are additionally reduced by 2286 per year


In [37]:
print('The reduction in orders after 1 year would be',round(np.sum(df['OrdersRed1']))+round(np.sum(df['OrdersRed2']))+round(np.sum(df['OrdersRed3'])), 'per year, which results in estimated cost savings of',(round(np.sum(df['OrdersRed1']))+round(np.sum(df['OrdersRed2']))+round(np.sum(df['OrdersRed3'])))*100, 'USD per year.'  )

The reduction in orders after 1 year would be 8833 per year, which results in estimated cost savings of 883300 USD per year.


In [28]:
print('If all stores achieve the target, the expected cost savings would be', np.sum(df['FinalOrderRed'])*100, 'USD per year.')

If all stores achieve the target, the expected cost savings would be 1124900.0 USD per year.


Using this model, purchase orders would be reduced by 3,711 orders in the first round (four-month period), purchase orders would be additionally reduced by 2,836 in the second round, and purchase orders would be additionally reduced by 2,286 orders in the third round. The estimated cost saving across after the first year is 883,300 USD per year. By the end of the year, three of the underachievers would have reached the target goal, and one more (Albuquerque) would shortly thereafter. The other two (Nashville and Austin) would have at least a few more rounds to go.

When all stores reach the target, the yearly cost savings are 1,124,900 USD compared to the current state.

Overall, Albuquerque, Nashville, Omaha, and Austin were the stores that struggled most with meeting the purchase order and supplier targets. To assist these stores in achieving the goals, they are matched in a mentor relationship with a successful store of similar size (merchandise purchased). Albuquerque is paired with Houston, Nashville is paired with Memphis, Omaha is paired with Milwaukee, and Austin is paired with El Paso. These mentors coach and encourage the weaker stores to achieve their targets ahead of time. (This is an example of a nonquantitative recommendation.)

As with any performance targets, these goals are current targets that should be reevaluated regularly. Ultimately, there is a reasonable range of suppliers and average merchandise per purchase order. Too low of either of these will inhibit the business from functioning optimally.

[[Back to TOC]](#Table-of-Contents)